src/test/feature/gtest-parallel

```diff
@@ -170,7 +170,7 @@ class FilterFormat:
       self.out.permanent_line("FAILED TESTS (%d/%d):"
                               % (len(self.failures), self.total_tests))
       for (binary, test) in self.failures:
-        self.out.permanent_line(" " + binary + ": " + test)
+        self.out.permanent_line(" " + binary + ": " + test + " [" + test_map[test] + "]")
```

Comment:
I suggest you to use format string for readability and correctness. For example, use `" %s: %s [ %s ]" % (binary, test, test_map[test])` here.